bigchaindb/common/transaction.py

```diff
@@ -1248,19 +1251,15 @@ def __str__(self):
         tx = Transaction._remove_signatures(self.to_dict())
         return Transaction._to_str(tx)
 
-    @classmethod
-    # TODO: Make this method more pretty
-    def from_dict(cls, tx_body):
-        """Transforms a Python dictionary to a Transaction object.
+    @staticmethod
+    def validate_structure(tx_body_orig):
+        """Validate the structure of a transaction body.
 
             Args:
                 tx_body (dict): The Transaction to be transformed.
-
-            Returns:
-                :class:`~bigchaindb.common.transaction.Transaction`
```

Comment:
Actually that's a fair point

![brother-cary-elwes-gnJgBlPgHtcnS](https://camo.githubusercontent.com/090027e9245c4d00a4653123a95ed54cf671bf25/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f676e4a67426c50674874636e532f67697068792e676966>)